python/lsst/obs/lsstSim/processEimage.py

```diff
@@ -25,7 +25,9 @@
 from lsst.pipe.tasks.processCcd import ProcessCcdTask
 from .eimageIsr import EimageIsrTask
 
+
 class ProcessEimageConfig(ProcessCcdTask.ConfigClass):
+
```

Comment:
This extra line looks wrong. And there's supposed to be a blank line after the class docstring.